src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpApplicationPreambleControllerTests.cs

```diff
@@ -74,7 +77,8 @@ public void Before_each_test()
                 _organisationApiClient.Object,
                 _usersApiClient.Object,
                 _applicationApiClient.Object,
-                _qnaApiClient.Object);
+                _qnaApiClient.Object,
+                _ukprnWhitelistValidator.Object);
```

Comment:
Have you checked that the unit tests still behave in the correct way? I can't see:  `_ukprnWhitelistValidator.SetUp( x=> x.IsWhitelistedUkprn(It.IsAny<int>)).ReturnsAsync(true);`

Is there going to be a unit test to check whitelisted UKPRNs?

Reply:
Test for the new functionality was created
Validation_error_is_triggered_if_UKPRN_is_not_in_whitelisted